bin.js

```diff
@@ -32,16 +36,23 @@ if (help || args.length === 0) {
   log('  -h, --help     Display this usage info')
   log('  -G, --no-glob  Do not expand glob patterns in arguments')
   log('  -g, --glob     Expand glob patterns in arguments (default)')
+  log('  -V, --verbose  Prints paths being deleted')
```

Comment:
The `rm` command supports `-v` (lowercase) instead of `-V`. We could support both.